src/views/partials/searchbar.hbs

```diff
@@ -1,6 +1,9 @@
-<div id="search">
-  <form class="search__form" action="/articles" method="POST">
-     <input class="search__input" type="text" name="search" placeholder="Search by keyword">
-     <input type="submit" value="Submit">
+<div id="search" class="w-100 tc">
+  <form class="mb4 " action="/articles" method="POST">
+     <input class="input__reset br4 pa2" type="text" name="search" placeholder="Search by keyword">
```

Comment:
Personally, I think this looks slightly better without the user-agent border and a tiny bit more padding horizontally:
`class="input__reset b--none br4 pv2 ph3"`
![image](https://user-images.githubusercontent.com/7805691/35098578-69ce1f74-fc5d-11e7-98bb-3d8c6fe5e5f1.png)
than how it is at the moment:
![image](https://user-images.githubusercontent.com/7805691/35098587-7a8ffabc-fc5d-11e7-9330-b651c069d146.png)

But it's not game-breaking!